cpp/src/io/parquet/reader_impl.hpp

```diff
@@ -130,10 +130,21 @@ class reader::impl {
                     bool uses_custom_row_bounds,
                     host_span<std::vector<size_type> const> row_group_indices);
 
+  /**
+   * @brief Create chunk information and start file reads
+   *
+   * @param row_groups_info vector of information about row groups to read
+   * @param num_rows  Maximum number of rows to read
+   * @return pair of boolean indicating if compressed chunks were found and a vector of futures for
+   * read completion
```

Comment:
```suggestion
   *         read completion
```

Reply:
I would like to push it out as well, but pre-commit did that format and changes it back to that.